Neeo binding will not load on M2 nightly

Please see https://community.openhab.org/t/neeo-remote-binding-transport/33408/390 and associated replies for the longer discussion.

When installing or starting OH with the Neeo binding and IO installed, system does not load properly.  This seems to cause many other bindings to fail at startup and the system is unusable. 
2019-06-01 14:55:56.258 [ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing ‘openhab-misc-neeo’

The only way this seems to work is to extract org.openhab.io.neeo-2.5.0.M1.jar and org.openhab.binding.neeo-2.5.0.M1.jar from the M1 kar, wait until OH has started completely, and then put the two files into the addons folder.  If the files are in the addons folder at start it causes the same effect.

--- a/bundles/org.openhab.io.neeo/src/main/feature/feature.xml
+++ b/bundles/org.openhab.io.neeo/src/main/feature/feature.xml
@@ -2,5 +2,8 @@
 <features name="org.openhab.io.neeo-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
     <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
 
-
+     <feature name="openhab-misc-neeo" description="NEEO Integration" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.io.neeo/${project.version}</bundle>
+    </feature>
 </features>

